src/benchmark_runner.cc

```diff
@@ -152,51 +160,52 @@ class BenchmarkRunner {
         perf_counters_measurement_ptr(perf_counters_measurement.IsValid()
                                           ? &perf_counters_measurement
                                           : nullptr) {
-    run_results.display_report_aggregates_only =
+    run_results->display_report_aggregates_only =
         (FLAGS_benchmark_report_aggregates_only ||
          FLAGS_benchmark_display_aggregates_only);
-    run_results.file_report_aggregates_only =
+    run_results->file_report_aggregates_only =
         FLAGS_benchmark_report_aggregates_only;
     if (b.aggregation_report_mode() != internal::ARM_Unspecified) {
-      run_results.display_report_aggregates_only =
+      run_results->display_report_aggregates_only =
           (b.aggregation_report_mode() &
            internal::ARM_DisplayReportAggregatesOnly);
-      run_results.file_report_aggregates_only =
-          (b.aggregation_report_mode() & internal::ARM_FileReportAggregatesOnly);
+      run_results->file_report_aggregates_only =
+          (b.aggregation_report_mode() &
+           internal::ARM_FileReportAggregatesOnly);
       CHECK(b.threads() == 1 || !perf_counters_measurement.IsValid())
           << "Perf counters are not supported in multi-threaded cases.\n";
       CHECK(FLAGS_benchmark_perf_counters.empty() ||
             perf_counters_measurement.IsValid())
           << "Perf counters were requested but could not be set up.";
     }
 
-    for (int repetition_num = 0; repetition_num < repeats; repetition_num++) {
+    for (size_t repetition_num = 0; repetition_num < repeats;
```

Comment:
no reason for this to change to `size_t`

Reply:
DONE